modules/progress.py

```diff
@@ -64,12 +66,16 @@ def finish_task(id_task):
         current_task = None
         current_task_step = ''
 
+    if id_task not in finished_tasks:

```

Comment:
Is there a list for failed tasks? Or the task id just not kept in record? And where the `id_task` being added into `finished_tasks`

Reply:
no, the failed task id it not recorded.
check line 70 to see how all task_ids are logged.